Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionConfigurationChecker.cs

```diff
@@ -13,7 +14,7 @@ namespace Microsoft.MixedReality.Toolkit.LeapMotion
     /// <summary>
     /// Class that checks if the Leap Motion Core assets are present and configures the project if they are.
     /// </summary>
-    [InitializeOnLoad]
```

Comment:
Nice!

We can also remove the exemption now too!

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/scripts/ci/validatecode.ps1#L370

Reply:
Removed leap from the exceptions